tests/e2e/e2e_setup_test.go

```diff
@@ -372,12 +372,19 @@ func (s *IntegrationTestSuite) runIBCRelayer() {
 	s.hermesResource, err = s.dkrPool.RunWithOptions(
 		&dockertest.RunOptions{
 			Name:       fmt.Sprintf("%s-%s-relayer", s.chainA.id, s.chainB.id),
-			Repository: "ghcr.io/cosmos/hermes-e2e",
-			Tag:        "latest",
+			Repository: "osmolabs/hermes",
```

Comment:
shouldn't this be `hermes-e2e`?

Reply:
I don't think so.

They called it `hermes-e2e` because it was a modified version of the official `hermes` to be able to run tests (mainly due to permission problems). 

WIth the changes made we can use the official  `hermes`  image to run tests saving us the time needed to build the `hermes-e2e` image.